src/schema/v2/jobs.ts

```diff
@@ -73,27 +117,27 @@ export const job: GraphQLFieldConfig<void, ResolverContext> = {
     id: { type: new GraphQLNonNull(GraphQLID) },
   },
   type: new GraphQLNonNull(jobType),
-  resolve: (_source, { id }, { jobLoader }) => {
-    return jobLoader(id)
+  resolve: async (_source, { id }) => {
+    const { results } = await ashby("jobPosting.info", {
+      body: { jobPostingId: id },
+    })
+    return results
```

Comment:
This approach feels totally fine for something like this! Graphql is made of being flexible, so this feels 👍

Reply:
Agreed, if there's no reason to go through DataLoader - feel free to just return a promise here, or resolve data further - it's all good.